pyvrp/cpp/Individual.cpp

```diff
@@ -207,8 +203,6 @@ Individual::Route::Route(ProblemData const &data, Visits const visits)
                       : 0;
 }
 
-bool Individual::Route::empty() const { return visits_.empty(); }
```

Comment:
Without this method things like `std::empty(route)` no longer work. `Individual::Route` is a container and asking whether a container is empty is not unusual.

Reply:
Also based on my assumption we would not have empty routes, I added them back.